yapapi_service_manager/service_wrapper.py

```diff
@@ -19,14 +19,14 @@ def status(self):
         if self.service is not None:
             service_state = self.service.state.value
             if service_state == 'terminated':
-                if self.stopped:
+                if self.stopped:  # pylint: disable=no-else-return
```

Comment:
Maybe we could put this in the `.pylintrc`?
Or maybe there are some cases when this makes more sense?

Reply:
I believe in most cases this rule makes sense but this function has quite complex conditional-branching so I decided to make a local exception.